pkgs/servers/sql/postgresql/ext/pgaudit.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, kerberos, openssl, postgresql }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, libkrb5, openssl, postgresql }:
```
`kerberos` is an alias, so this won't evaluate without them